xija/gui_fit/plots.py

```diff
@@ -160,6 +160,59 @@ def clearLayout(layout):
                 clearLayout(item.layout())
 
 
+def annotate_limits(limits, ax, dir='h'):
+    if len(limits) == 0:
+        return []
+    lines = []
+    draw_line = getattr(ax, f'ax{dir}line')
+    if 'planning.data_quality.high.acisi' in limits:
```

Comment:
These could all be factored into an enclosed function `draw_line(limit_name, linestyle, color)` which has `limits` in scope so you just have a series of calls like:
```
for limit_name, linestyle, color in [
    ('planning.data_quality.high.acisi', '-.', 'blue'),
    (...)]:
    draw_line(limit_name, linestyle, color)
```